lib/scss/components/card.component.scss

```
@@ -2,3 +2,18 @@
 @import '../jumpstart_ui.config';
 @import 'decanter-no-markup';
 @import '~decanter/scss/components/card/index';
+
+.su-card {
+  .media.video {
```

```suggestion
  .media {
    &.video {
```
https://css-tricks.com/the-sass-ampersand/#article-header-id-1